archive/tar_opts.go

```diff
@@ -89,7 +90,18 @@ type WriteDiffOptions struct {
 	ParentLayers []string // Windows needs the full list of parent layers
 
 	writeDiffFunc func(context.Context, io.Writer, string, string, WriteDiffOptions) error
+
+	// SourceDateEpoch specifies the SOURCE_DATE_EPOCH without touching the env vars.
```

Comment:
nit on the phrasing here (and in the other places it's used): While SOURCE_DATE_EPOCH is usually an environment variable, I think we can describe its purpose rather than the usual mechanism.  I would probably rephrase this as "SourceDateEpoch specifies the timestamp used for whiteouts to provide control for reproducibility" and maybe link to https://reproducible-builds.org/docs/source-date-epoch/ as background information.

Reply:
Fixed